scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -259,7 +272,7 @@ class ScanCodeResultParserTest : WordSpec({
             val result = readJsonFile(resultFile)
 
             val actualFindings = generateSummary(Instant.now(), Instant.now(), SpdxConstants.NONE, result)
-                    .licenseFindings
+                .licenseFindings
```

Comment:
There's again this unrelated formatting change.

Reply:
This time it is intended, see the first commit.